Dockerfile-alpine.template

```diff
@@ -8,6 +8,7 @@ COPY requirements.txt /tmp/
 RUN set -ex \
     && apk add --no-cache --virtual .build-deps \
        gcc \
+       g++ \
```

Comment:
Saw some build failures due to a missing `cc1plus` build dependency. This is installed as part of `g++`, which is apparently now required for building `gevent`. Installing `g++` doesn't add to image size since we uninstall build dependencies later on in the build.

We could also just install `g++` instead of `gcc` since `gcc` is a dependency of `g++`. I erred on the side of being more explicit to indicate that we specifically need C _and_ C++ build dependencies instead of just installing the kitchen sink.